lib/PSR/Streaming.hs

```diff
@@ -159,3 +162,43 @@ streamChainSyncEvents ::
     Stream IO ChainSyncEvent
 streamChainSyncEvents conn points =
     Stream.fromCallback (void . forkIO . subscribeToChainSyncEvents conn points)
+
+streamBlocks :: CM.ConfigMap -> [C.ChainPoint] -> Stream IO (C.ChainPoint, Block)
+streamBlocks CM.ConfigMap{..} points =
+    streamChainSyncEvents cmLocalNodeConn points
+        & Stream.filter (not . isByron)
+        & fmap getEventBlock
```

Comment:
A TODO for later, we can probably filter here to remove any block that doesn't reference any scripts we care about.

Reply:
I'll note this TODO down. We filter the transactions anyway.
Filtering the blocks will involve going through all the transactions again. Not sure if it'll give any significant performance improvements.